README.md

```diff
@@ -41,20 +58,26 @@ At the end of the script you will see a URL to a download page. Save this link.
 
 ### Option 2: Deploy from your local machine
 
+> Note: Deployment from MacOS has been verified. Local deployment from Windows might require some additional setup. Please refer to the additional resources.
```

Comment:
Can we delete this?

Reply:
I have had some issues with the deploy script on windows. Some further work will be required to iron out all the issues.